pom.xml

```diff
@@ -43,6 +43,12 @@
             <version>4.0.0</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>RELEASE</version>
```

Comment:
This is not how a dependency version is defined. It should probably be something like `16.0.2`

Reply:
It was generated by Inteligj IDEA automaticly.